controllers/grid/users/reviewer/AuthorReviewerGridCellProvider.php

```diff
@@ -75,7 +75,13 @@ public function getTemplateVarsFromRowColumn($row, $column)
                 return ['label' => __($element->getReviewMethodKey())];
 
             case 'considered':
-                return ['label' => $this->_getStatusText($this->getCellState($row, $column), $row)];
+                $statusText = $this->_getStatusText($this->getCellState($row, $column), $row);
+                $reviewAssignment = $row->getData();
+                $competingInterests = $reviewAssignment->getCompetingInterests();
+                if ($competingInterests) {
+                    $statusText .= '<span class="details">' . __('reviewer.competingInterests') . '</span>';
```

Comment:
@nibou230 Does it look very bad if we remove the `<span>`?
I just think it's better to not bring HTML into the PHP side (even though there are already some cases on the codebase).

If it doesn't look very nice, then I think it's ok to leave it this way (all these `Grid` classes are supposed to be removed sooner or later) 👌

Reply:
Hi @jonasraoni, it looks alright. See the image below:

![image](https://github.com/pkp/pkp-lib/assets/31410053/e79317b0-0f5a-4aa6-9d5c-0bba8bfb3d5a)

As you said, those grids are already heavily using HTML inside the PHP code. To make it look inline with other labels, I also propose to let the `<span>` in place for now if it's good for you!

Reply:
Yeah, I think it's better to leave the `<span>`. I'll merge 😁